### PR TITLE
Framework agreement small fixes, fill in changelog

### PR DESCRIPTION
**Pull request by jpmckinney**
- guidance/map/framework_agreements: Fix typos and style booleans correctly
- guidance/map/organization_classifications: Style booleans correctly

### CODE REVIEW

**Comment by jpmckinney** [comment]
~~We need to discuss https://github.com/open-contracting/standard/issues/1301#issuecomment-905886947 before merging.~~

#1301 will take longer to agree on what to do about procurement method, so we can do that in another PR.

**Comment by jpmckinney** [comment]
I noticed several PRs weren't mentioned in the changelog. There's now a script to find them.

**Review by yolile — APPROVED** [review]

File: docs/history/changelog.md [hunk 1]
```diff
@@ -8,40 +8,61 @@ Per the [normative and non-normative content and changes policy](https://docs.go
...
-  * [#1013](https://github.com/open-contracting/standard/pull/1013) Replace guidance on publication levels.
-  * [#1016](https://github.com/open-contracting/standard/pull/1016) Add links to OCDS profiles and linked standards.
-  * [#1150](https://github.com/open-contracting/standard/pull/1150) Align language in Build phase with language in Map phase.
-  * [#1193](https://github.com/open-contracting/standard/pull/1193) Removed language distinguishing "core", "community", and "local" extensions.
-  * [#1337](https://github.com/open-contracting/standard/pull/1337) Improve the structure of the Map page.
-  * [#1338](https://github.com/open-contracting/standard/pull/1338) Improve the structure of the worked examples on the Map page.
-  * [#1332](https://github.com/open-contracting/standard/pull/1332) Improve guidance on framework agreements.
   * [#1386](https://github.com/open-contracting/standard/pull/1386) Add link to OCDS Implementation Checklist.
+  * Map:
+    * Add examples [#1338](https://github.com/open-contracting/standard/pull/1338):
```
Comment (yolile) [inline docs/history/changelog.md:18]: @jpmckinney do we want to change the order now?

Comment (jpmckinney) [reply docs/history/changelog.md:18]: What would we change it to? Right now it's just in PR order.

Comment (jpmckinney) [reply docs/history/changelog.md:18]: Anyway, I'll merge as approved. We can change the log in another PR.

**Pull request merged**